.github/workflows/ci.yml

```diff
@@ -11,10 +11,10 @@ jobs:
     strategy:
       matrix:
         nominatim:
-          - version: 3.7
+          - version: "4.0"
+            update_command: docker exec -i nominatim sudo -u nominatim nominatim replication --project-dir /nominatim --once
+          - version: "3.7"
             update_command: docker exec -i nominatim sudo -u nominatim nominatim replication --project-dir /nominatim --once
-          - version: 3.6
```

Comment:
Do we want to remove support for 3.6? That's fine with me tbh, but then we should also reflect the change in the `README.md` and move 3.6 it to `deprecated tags`.

Reply:
Good point. I will fix this.